CMake/FindYARA.cmake

```diff
@@ -0,0 +1,51 @@
+# - Find the YARA library
+# This module defines
+#  YARA_INCLUDE_DIR, path to yara.h, etc.
+#  YARA_LIBRARIES, the libraries required to use YARA.
+#  YARA_FOUND, If false, do not try to use YARA.
+# also defined, but not for general use are
+# YARA_LIBRARY, where to find the YARA library.
+
+# Apple readline does not support readline hooks
+# So we look for another one by default
+IF(APPLE)
+  FIND_PATH(YARA_INCLUDE_DIR NAMES yara.h PATHS
+    /sw/include
+    /opt/local/include
+    /opt/include
+    /usr/local/include
+    /usr/include/
+    NO_DEFAULT_PATH)
+ENDIF(APPLE)
+FIND_PATH(YARA_INCLUDE_DIR NAMES yara.h)
+
+IF(APPLE)
+  FIND_LIBRARY(YARA_LIBRARY NAMES yara PATHS
+    /sw/lib
```

Comment:
This is a copy from the readline piece of the build which I modified accordingly. I never touched any of the search paths though. My personal opinion is that it doesn't hurt to have it there in case someone installed YARA there.

If you want me to fix it I can do so in a separate pull request which sweeps the tree of these kinds of problems.